.env.development

```diff
@@ -1 +1,2 @@
-REACT_APP_API_ROOT=/
+# Root should be "/" if you have the backend running locally.
```

Comment:
You probably thought of this already but can we automate this in any way? Adding another env and/or adding a run script in package.json for running locally?

Reply:
Yeah I did. I'll see if I can make a solution were the developer has a simple `.env` that overrides whatever is in `.env.development`.

Reply:
There was actually already a solution: use `.env.development.local`.